packages/components/src/InputEmail/InputEmail.types.ts

```diff
@@ -0,0 +1,60 @@
+import { Clearable } from "@jobber/hooks";
+import { CommonFormFieldProps, FormFieldProps } from "../FormField";
+
+export type InputEmailLegacyProps = CommonFormFieldProps &
+  Pick<
+    FormFieldProps,
+    "maxLength" | "readonly" | "validations" | "defaultValue"
+  >;
+
+export const validationMessage = "Please enter a valid email";
+
+export type InputEmailVersion = 1 | 2 | undefined;
+
+/**
+ * Experimental version 2 of the InputEmail component.
+ * Do not use unless you have talked with Atlantis first.
+ */
+export interface InputEmailRebuiltProps
+  extends Omit<
+    React.InputHTMLAttributes<HTMLInputElement>,
+    | "onChange"
+    | "onBlur"
+    | "maxLength"
```

Comment:
This omit isn't needed since `input` takes a `maxLength` that is the same as the one you have defined below